modules/tests/jvm/src/test/scala/gsp/math/skycalc/ImprovedSkyCalcSpecJVM.scala

```diff
@@ -8,8 +8,6 @@ import weaver._
 import weaver.scalacheck._
```

Comment:
Is there a js version of this test?

Reply:
Not this test, since it compares the output to the one from the original OCS version. There is another test added by your suggestion that cross compiles and just checks a couple of known values.